packages/docusaurus-theme-redoc/src/theme/Redoc/Redoc.tsx

```diff
@@ -20,7 +16,7 @@ function Redoc(props: RedocProps): JSX.Element {
   const { className, optionsOverrides, url, themeId } = props;
   const { options } = useSpecOptions(themeId, optionsOverrides);
 
-  if (getIsExternalUrl(url)) {
```

Comment:
this is going to completely disable SSR for anyone using url, if that is what you want then we can just add that as an option I think rather than changing behavior for existing users.

We won't need `downloadSpecUrl` option, just a `ssr` option, by default `true` but which can be disabled to always use client side rendering and skip the bundling logic

Reply:
Allright I will try to do that change